static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -95,7 +96,7 @@ $ tree .dvc/cache
     └── 0b40427ee0998e9802335d98f08cd98f
 ```
 
-The cache file with `.dir` extension is a special text file that stores the
+The cache file with `.dir` extension is a special text file that records the
```

Comment:
records sounds like an action - like audio recording ... it stores that data in the same way as databases store data

Reply:
How about "contains"? I just don't want anyone to think that outputs themselves are **stored** in .dir files.